service/src/test/java/greencity/service/HabitAssignServiceImplTest.java

```diff
@@ -287,12 +289,36 @@ void unenrollHabitThrowWrongIdException() {
     }
 
     @Test
-    void deleteHabitAssign() {
+    void changeStatusHabitAssign() {
         HabitAssign habitAssign = ModelUtils.getHabitAssign();
-        when(habitAssignRepo.findByUserIdAndHabitId(1L, 1L)).thenReturn(Optional.ofNullable(habitAssign));
-        assert habitAssign != null;
-        habitAssignService.deleteHabitAssign(1L, 1L);
-        verify(habitAssignRepo).delete(habitAssign);
+        habitAssign.getHabit().setHabitAssigns(generateHabitAssignList());
+        when(habitAssignRepo.findListByUserIdAndHabitIdAndStatusIsInProgress(1L, 1L))
+            .thenReturn(habitAssign.getHabit().getHabitAssigns());
+        habitAssignService.cancelAllHabitAssign(1L, 1L);
+        Mockito.verify(userShoppingListItemRepo).deleteByShoppingListItemsByHabitAssignId(1L);
+        assert HabitAssignStatus.INPROGRESS != habitAssign.getHabit().getHabitAssigns().get(0).getStatus();
+        verify(habitAssignRepo, times(2)).save(any());
```

Comment:
we [change](https://github.com/ita-social-projects/GreenCity/pull/4865/files#diff-f0d395cd384c0467bf1f8b70068c9c75d29428d84a9f20c4401545f84b17e5b4R885) the status to `CANCELED` before saving. Could you please verify that we call `save` method with the expected habit (status is `CANCELED`)